out/notebooks/provider.ts

```diff
@@ -64,6 +91,11 @@ export class CellResultCountStatusBarProvider implements vscode.NotebookCellStat
                             "arguments": [cell]
                         }
                     });
+                    // earliest and latest time pickers are only supported for SPL2 at the moment
```

Comment:
Why is this done in 2 places?

Reply:
One is for the state when the job hasn't been run yet and the other is for the state where there are outputs from the job. There are a different set of icons for each.

Before:
<img width="925" alt="image" src="https://github.com/splunk/vscode-extension-splunk/assets/4960530/137d7fff-58c3-40dc-ab59-28991df045f2">

After (with job details):
<img width="846" alt="image" src="https://github.com/splunk/vscode-extension-splunk/assets/4960530/b1f53168-75c7-4f6f-a894-64a7f0bc00fa">